packages/design-system-react/package.json

```diff
@@ -35,10 +35,13 @@
     "dist/"
   ],
   "scripts": {
-    "build": "ts-bridge --project tsconfig.build.json --verbose --clean --no-references",
+    "build": "yarn generate:icons && ts-bridge --project tsconfig.build.json --verbose --clean --no-references",
     "changelog:update": "../../scripts/update-changelog.sh @metamask/design-system-react",
     "changelog:validate": "../../scripts/validate-changelog.sh @metamask/design-system-react",
+    "clean:icons": "rimraf src/components/icon/icons",
     "create-component": "ts-node scripts/create-component",
+    "generate:icons": "yarn clean:icons && svgr --config-file src/components/icon/.svgrrc.js -d src/components/icon/icons src/components/icon/assets/*.svg && yarn generate:icons:index",
+    "generate:icons:index": "ts-node scripts/generate-icons-index.ts",
```

Comment:
These scripts manage our icon generation pipeline:
- clean:icons: Removes previously generated icon components to ensure a clean slate
- generate:icons: Main icon generation script that:
1. Cleans existing icons
2. Uses SVGR to convert SVG files from assets/ into React components
3. Generates an index file for easy importing
- generate:icons:index: Creates an index file that exports all generated icon components for convenient access